.scripts/package.sh

```diff
@@ -329,7 +329,7 @@ if [[ $latest != $current || $debug ]]; then
     mv "$scratch/$package" "$package"
 
     # Skips deploy
-    if [[ $skip_release ]]; then echo "Done."; exit 0; fi
```

Comment:
@evandcoleman ran this locally since the GH Action is broken. Will leave this commented while debugging